ngraph/test/runtime/interpreter/int_executable.hpp

```diff
@@ -1200,8 +1200,10 @@ class INTERPRETER_BACKEND_API ngraph::runtime::interpreter::INTExecutable : publ
         case OP_TYPEID::Round:
         {
             size_t element_count = shape_size(node.get_output_shape(0));
-            reference::round<T>(
-                args[0]->get_data_ptr<const T>(), out[0]->get_data_ptr<T>(), element_count);
+            reference::round<T>(args[0]->get_data_ptr<const T>(),
```

Comment:
You modified the test for v0::Round. But you need to add v5::Round tests. Please, refer to https://github.com/openvinotoolkit/openvino/pull/2587 for example how to do this.

Reply:
Done.